lib/models/movie.js

```diff
@@ -7,7 +7,7 @@ module.exports = Bookshelf.Model.extend({
   serialize: function () {
     return {
       id: this.get('id'),
-      title: this.get('title'),
+      title: this.get('name'),
```

Comment:
Why are you serializing `name` as `title`?

Reply:
Because in the db we have altered the table, s.t. there's a name column which stores title of the movie. It's one of the exercises of miyagi